exercises/rational-numbers/src/test/java/RationalTest.java

```diff
@@ -0,0 +1,293 @@
+import org.junit.Ignore;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class RationalTest {
+
+    // Helper methods
+
+    private static final double DOUBLE_EQUALITY_TOLERANCE = 1e-15;
+
+    private void assertDoublesEqual(double x, double y) {
+        assertEquals(x, y, DOUBLE_EQUALITY_TOLERANCE);
+    }
+
+    private void assertRationalsEqual(Rational r1, Rational r2) {
+        assertEquals(r1.getNumerator(), r2.getNumerator());
+        assertEquals(r1.getDenominator(), r2.getDenominator());
+    }
+
+    // Tests
+
+    @Test
+    public void testAddTwoPositiveRationalNumbers() {
+        Rational expected = new Rational(7, 6);
+        Rational actual = new Rational(1, 2).add(new Rational(2, 3));
+        assertRationalsEqual(expected, actual);
+    }
+
+    @Test
+    @Ignore
```

Comment:
As described in our [POLICIES doc](https://github.com/exercism/java/blob/master/POLICIES.md#ignore-noninitial-testshttps://github.com/exercism/java/blob/master/POLICIES.md#ignore-noninitial-tests) this should be `@Ignore("Remove to run test")` and it should be above the `@Test` annotation 🙂